.changeset/config.json

```
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.2/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "react-native-async-storage/async-storage" }
+  ],
+  "commit": ["@changesets/cli/commit", { "skipCI": false }],
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
```

Do we need to ignore eslint-config and website?
```suggestion
  "ignore": [
    "@react-native-async-storage/eslint-config",
    "async-storage-website"
  ]
```